<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxiesConfigWidget</class>
 <widget class="QWidget" name="ProxiesConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="ProxiesList_"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Target</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="buddy">
           <cstring>TargetHost_</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="buddy">
           <cstring>TargetPort_</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="TargetPort_">
          <property name="specialValueText">
           <string>any</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Protocols:</string>
          </property>
          <property name="buddy">
           <cstring>TargetProto_</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="TargetProto_">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string notr="true">http</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">ftp</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="TargetHost_">
          <property name="placeholderText">
           <string>Host name or regular expression...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Proxy</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Type:</string>
          </property>
          <property name="buddy">
           <cstring>ProxyType_</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="ProxyType_">
          <item>
           <property name="text">
            <string notr="true">SOCKS5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">HTTP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">Caching HTTP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">Caching FTP</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="buddy">
           <cstring>ProxyHost_</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ProxyHost_"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="buddy">
           <cstring>ProxyPort_</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>User:</string>
          </property>
          <property name="buddy">
           <cstring>ProxyUser_</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ProxyUser_"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="buddy">
           <cstring>ProxyPassword_</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="ProxyPassword_"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="ProxyPort_">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="AddProxyButton_">
     <property name="text">
      <string>Add proxy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TargetHost_</tabstop>
  <tabstop>TargetPort_</tabstop>
  <tabstop>TargetProto_</tabstop>
  <tabstop>ProxyType_</tabstop>
  <tabstop>ProxyHost_</tabstop>
  <tabstop>ProxyPort_</tabstop>
  <tabstop>ProxyUser_</tabstop>
  <tabstop>ProxyPassword_</tabstop>
  <tabstop>AddProxyButton_</tabstop>
  <tabstop>ProxiesList_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>TargetHost_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddProxyButton_</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProxyHost_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddProxyButton_</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProxyUser_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddProxyButton_</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProxyPassword_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddProxyButton_</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
